sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -0,0 +1,188 @@
+    reason?: string;
+}
+
+// @public
+export interface GetAuthenticationTokenOptions {
+    claims?: {
+        [key: string]: string[];
+    };
+    userId?: string;
+}
+
+// @public
+export interface GetAuthenticationTokenResponse {
+    baseUrl: string;
+    token: string;
+    url: string;
+}
+
+// @public
+export interface GroupAddConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupAddUserOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupAdminClientOptions extends PipelineOptions {
+}
+
+// @public
+export interface GroupHasUserOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupRemoveConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupRemoveUserOptions extends OperationOptions {
+}
+
+// @public
+export interface GroupSendTextToAllOptions extends OperationOptions {
+    // (undocumented)
+    contentType: "text/plain";
+    excludedConnections?: string[];
+}
+
+// @public
+export interface GroupSendToAllOptions extends OperationOptions {
+    excludedConnections?: string[];
+}
+
+// @public
+export interface HasConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface HubAdminClientOptions extends PipelineOptions {
+}
+
+// @public (undocumented)
+export interface HubGrantPermissionOptions extends OperationOptions {
+    targetName?: string;
+}
+
+// @public
+export interface HubHasGroupOptions extends OperationOptions {
+}
+
+// @public
+export interface HubHasUserOptions extends OperationOptions {
+}
+
+// @public
+export interface HubRemoveUserFromAllGroupsOptions extends CloseConnectionOptions {
+}
+
+// @public (undocumented)
+export interface HubRevokePermissionOptions extends OperationOptions {
+    targetName?: string;
+}
+
+// @public
+export interface HubSendTextToAllOptions extends HubSendToAllOptions {
+    // (undocumented)
+    contentType: "text/plain";
+}
+
+// @public
+export interface HubSendTextToConnectionOptions extends HubSendToConnectionOptions {
+    // (undocumented)
+    contentType: "text/plain";
+}
+
+// @public
+export interface HubSendTextToUserOptions extends HubSendToUserOptions {
+    // (undocumented)
+    contentType: "text/plain";
+}
+
+// @public
+export interface HubSendToAllOptions extends OperationOptions {
+    excludedConnections?: string[];
+}
+
+// @public
+export interface HubSendToConnectionOptions extends OperationOptions {
+}
+
+// @public
+export interface HubSendToUserOptions extends OperationOptions {
+}
+
+// @public
+export type JSONTypes = string | number | boolean | object;
+
+// @public (undocumented)
+export type Permission = "joinLeaveGroup" | "sendToGroup";
+
+// @public (undocumented)
+export interface WebPubSubGroup {
+    addConnection(connectionId: string, options?: GroupAddConnectionOptions): Promise<RestResponse>;
+    addUser(username: string, options?: GroupAddUserOptions): Promise<RestResponse>;
```

Comment:
Are we going to get in trouble returning `RestResponse` from `core-http` if we switch to `core-rest-pipeline`? IIRC they're not the same shape

Reply:
There's a work item to adopt core-rest-pipeline, unfortunately there wasn't time to do it before the preview. I worry about not returning status codes generally since they're used a bit in this service. I would take the break, though.